modoboa/core/password_hashers/__init__.py

```diff
@@ -29,3 +34,26 @@ def get_password_hasher(scheme):
     except KeyError:
         hasher = PLAINHasher
     return hasher
+
+def get_dovecot_schemes():
+    """Return schemes supported by dovecot"""
+    supported_schemes = ["{PLAIN}"]
+    dpath = None
+    code, output = exec_cmd("which doveadm")
+    if not code:
+        dpath = force_text(output).strip()
+    else:
+        known_paths = ("/usr/bin/doveadm", "/usr/local/bin/doveadm")
```

Comment:
I think there is another place in the code where we look for doveadm path. I guess a new function would be welcome? (maybe in lib/sysutils.py)